docs/pages/desktop-access/active-directory-manual.mdx

```diff
@@ -33,21 +34,31 @@ This guide requires you to have:
 
 - Access to a Domain Controller
 
-- An existing Teleport cluster with one of the following versions:
-
-  **Open Source or Enterprise:** version 8.0 or newer
-
-  **Teleport Cloud:** version 9.0 or newer
-
-  See [Teleport Getting Started](../try-out-teleport/introduction.mdx) if you're new to Teleport.
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+  <Details
+    title="Version warning"
+    opened={true}
+    scope={["oss", "enterprise"]}
+    scopeOnly={true}
+    min="8.0"
+  >
+    Desktop Access is available in starting from Teleport `8`.
+  </Details>
+
+  <Details
```

Comment:
We can probably remove both of these `<Details>` blocks now, since v8 and v9 are long out of support.